lib/node_modules/@stdlib/stats/base/dists/binomial/cdf/README.md

```diff
@@ -144,23 +144,18 @@ y = mycdf( 1.0 );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var logEachMap = require( '@stdlib/console/log-each-map' );
 var cdf = require( '@stdlib/stats/base/dists/binomial/cdf' );
 
-var i;
-var n;
-var p;
-var x;
-var y;
-
-for ( i = 0; i < 10; i++ ) {
-    x = randu() * 20.0;
-    n = round( randu() * 100.0 );
-    p = randu();
-    y = cdf( x, n, p );
-    console.log( 'x: %d, n: %d, p: %d, F(x;n,p): %d', x.toFixed( 4 ), n, p.toFixed( 4 ), y.toFixed( 4 ) );
-}
+var opts = {
+    'dtype': 'float64'
+};
+var x = discreteUniform( 10, 0, 20, opts );
+var n = discreteUniform( 10, 0, 100, opts );
+var p = discreteUniform( 10, 0, 1, opts );
```

Comment:
Same comments as elsewhere. `p` is a probability. When authoring these examples, be sure to read the JSDoc of the JavaScript implementations and the README, as those provide indications as to expected value ranges.